src/utils/logger.js

```diff
@@ -0,0 +1,32 @@
+require('dotenv').config();
+const moment = require('moment');
+const winston = require('winston');
+
+var config = winston.config;
```

Comment:
what is defined in winston.config?

Reply:
use winston.config to set the colorized console log